packages/components/inputs/radio-input/src/radio-option.tsx

```diff
@@ -36,6 +85,7 @@ const Option = (props) => {
         css={accessibleHiddenInputStyles}
         id={props.id}
         name={props.name}
+        // @ts-ignore
```

Comment:
Why do we need to ignore next line?

Reply:
Could we use somehting like this instead?

Reply:
Good idea. Thanks.